### DISCUSSION TITLE
AA android auto not working properly.

### DISCUSSION THREAD

**Discussion by maurilucky** [osmandapp/OsmAnd#18890] [Q&A]
Device : OP4C2DL1
Brand : OPPO
Manufacturer : OPPO
Model : CPH2009
Product : CPH2009EEA
Build : CPH2009_11_F.11
Version : 12
App Version : OsmAnd+
Apk Version : 4.6.8 4608 

Hi android auto not working properly.
The apps run but after you block the screen on the phone the position will be lost/ not updated

I tried with two different phone (one plus nord 2) on two different cars (wireless aliexpress carplay screen for motorbike and toyota yaris cabled ) and both of them never work. Google maps work perfectly.

I check the permission and also tried to change the position retrieving from google play service to API but nothing changed.

Thanks for the support.

**Comment by maurilucky** [comment]
up

**Reply by DmitryAlexei** [reply]
Have you tried to disable battery optimization for OsmAnd? 
You can also try hints from docs (see https://osmand.net/docs/user/troubleshooting/android_auto#location-delays-anr-crashes )

**Comment by maurilucky** [comment]
Hi Dmitry,
 thanks for your reply.
I already tried everything on phone settings side.
But It's look like the app will be killed by the task manager every time I
block the screen.

Any advice to solve this issues?



Il gio 18 gen 2024, 18:28 DmitryAlexei ***@***.***> ha
scritto:
